examples/json_test/json_test.c

```diff
@@ -0,0 +1,91 @@
+/* Very basic example that just demonstrates we can run at all!
```

Comment:
Another minor tweak, this comment is from the original example.

Please add a short header comment & copyright notice for the example being public domain/copyright. Generally the examples are marked public domain / CC0 to make them easy to reuse.